components/infobox/wikis/freefire/infobox_team_custom.lua

```diff
@@ -0,0 +1,77 @@
+---
+-- @Liquipedia
+-- wiki=freefire
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Template = require('Module:Template')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Team = Lua.import('Module:Infobox/Team', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomTeam = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _team
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	_team = team
+	team.createWidgetInjector = CustomTeam.createWidgetInjector
+	team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	return team:createInfobox(frame)
+end
+
+function CustomTeam:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'staff' then
+		return {
+			Cell{name = 'Founders',	content = {_team.args.founders}},
+			Cell{name = 'CEO', content = {_team.args.ceo}},
+			widgets[4],
+			widgets[5],
+			widgets[1],
+			Cell{name = 'Analysts', content = {_team.args.analysts}},
+		}
```

Comment:
```suggestion
		table.insert(widgets, 1, Cell{name = 'Founders',	content = {_team.args.founders}})
		table.insert(widgets, 2, Cell{name = 'CEO', content = {_team.args.ceo}})
		table.insert(widgets, Cell{name = 'Analysts', content = {_team.args.analysts}})
```
Probably better to do this, otherwise if we add a new one on commons it might all break down. (Though I think I might have done this on some wiki implementation 😨)